### DISCUSSION TITLE
USD/KRW 실시간 싱크가 안맞는 경우가 있나요?

### DISCUSSION THREAD

**Discussion by devmjun** [FinanceData/FinanceDataReader#199] [Q&A] [answered]
안녕하세요. 

USD/KRW 값이 실시간 싱크가 안맞는 경우가 있나요? 
안맞는다면 어떻게 해결할 수 있나요?

**Comment by FinanceData** [comment, marked as answer]
FinanceDataReader는 yahoo finance, naver finance 웹사이트를 크롤링(조회)하기 때문에, 해당 웹 사이트에서 제공하는 지연 만큼을 그대로 가지게 됩니다.
